CMakeLists.txt

```diff
@@ -151,7 +151,14 @@ endif()
 set(CMK_REPLAYSYSTEM ${REPLAY})
 
 if(${TARGET} STREQUAL "all-test")
-  message(WARNING "Setting TARGET to LIBS instead of all-test.")
+  add_custom_target(all-test ALL COMMAND make -C ${CMAKE_BINARY_DIR}/tmp all-test)
```

Comment:
Does this have the drawback that it forces the dependent make to run as `-j1`?

Reply:
No, this will run in parallel.